core/jreleaser-engine/src/test/java/org/jreleaser/engine/init/InitTest.java

```diff
@@ -0,0 +1,28 @@
+package org.jreleaser.engine.init;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class InitTest {
+    @Test
+    void jsonSyntaxIsCorrect(@TempDir Path outputDirectory) {
+        // given:
+        String format = "json";
+
+        // when:
+        Init.execute(null, format, false, outputDirectory);
```

Comment:
Do not pass `null` as first argument. Rather use an existing implementation. I believe there's one used by SDK projects.